client/packages/common/src/intl/locales/en/app.json

```diff
@@ -59,5 +59,6 @@
   "button.login": "Log in",
   "error.login": "Invalid username or password",
   "select-store": "Select store",
-  "select-report": "Select Template"
+  "select-report": "Select Template",
+  "error.finding-servers": "Problem finding local network servers"
```

Comment:
Although I have no idea how language would change for the client, I didn't want to hard code strings